TransferMethod/Tests/SelectTransferMethodTypePresenterTests.swift

```diff
@@ -11,18 +11,18 @@ class SelectTransferMethodTypePresenterTests: XCTestCase {
     private let mockView = MockSelectTransferMethodTypeView()
```

Comment:
rename MockSelectTransferMethodTypeView to SelectTransferMethodTypeViewMock

Reply:
fixed